modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -16548,6 +16574,183 @@ private void _testFilterObjectEntriesByRelatedSystemObjectEntriesFields(
 		Assert.assertEquals("BAD_REQUEST", jsonObject.getString("status"));
 	}
 
+	private void _testGetCommentsEndpoints(
+			long groupId, ObjectDefinition objectDefinition)
+		throws Exception {
+
+		_enableComments(objectDefinition);
+
+		ObjectEntry objectEntry = ObjectEntryTestUtil.addObjectEntry(
+			objectDefinition, _OBJECT_FIELD_NAME_1, _OBJECT_FIELD_VALUE_1);
+
+		// Parent Comment
+
+		String endpoint = StringBundler.concat(
+			_getEndpoint(objectDefinition, groupId),
+			"/by-external-reference-code/",
+			objectEntry.getExternalReferenceCode(), "/comments");
+
+		String externalReferenceCode = RandomTestUtil.randomString();
+		String text = RandomTestUtil.randomString();
+
+		JSONObject commentJSONObject1 = _postComment(
+			endpoint, externalReferenceCode, text);
+
+		JSONObject commentJSONObject2 = _postComment(
+			endpoint, RandomTestUtil.randomString(),
+			RandomTestUtil.randomString());
+
+		JSONObject getCommentJSONObject = HTTPTestUtil.invokeToJSONObject(
```

Comment:
This variable name is not good. 
Should this return a list of comments? Or is the list inside the JSONObject? Something like this:
`JSONArray itemsJSONArray = jsonObject.getJSONArray("items");`
Did you check `ObjectEntryResourceTest#_assertPagination` example?